features/docs/gegeven-stap-definities-expressief.feature

```diff
@@ -0,0 +1,142 @@
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10001,0,0,P,000000036,P1                    |
+      |      | ouder-1-1    | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10001,0,0,1,000000012,P2                    |
+      |      | ouder-2-1    | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam) VALUES($1,$2,$3,$4,$5,$6)                                                                        | 10001,0,0,2,000000024,P3                    |
+
+  Scenario: de persoon '[aanduiding]' met burgerservicenummer '[bsn]' is geadopteerd
+    Gegeven de persoon 'P1' met burgerservicenummer '000000036'
+    * is geadopteerd
+      Dan zijn de gegenereerde SQL statements
+      | stap | categorie    | text                                                                                                                                                  | values                          |
+      | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING * | 0                               |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam,akte_nr) VALUES($1,$2,$3,$4,$5,$6,$7)         | 9999,0,0,P,000000036,P1,1AQ0100 |
+
+  Scenario: de persoon '[aanduiding]' met burgerservicenummer '[bsn]' heeft ouder 1 en gegevens van de ouder zijn gewijzigd
+    Gegeven de persoon 'P1' met burgerservicenummer '000000036'
+    * heeft een ouder 1 'P2' met burgerservicenummer '000000012'
+      En zijn de volgende gegevens van ouder 'P2' gewijzigd
+      | datum ingang familierechtelijke betrekking (62.10) |
+      | 20200101                                           |
+      Dan zijn de gegenereerde SQL statements
+      | stap | categorie    | text                                                                                                                                                             | values                           |
+      | 1    | inschrijving | INSERT INTO public.lo3_pl(pl_id,mutatie_dt,geheim_ind) VALUES((SELECT COALESCE(MAX(pl_id), 0)+1 FROM public.lo3_pl),current_timestamp,$1) RETURNING *            | 0                                |
+      |      | persoon      | INSERT INTO public.lo3_pl_persoon(pl_id,stapel_nr,volg_nr,persoon_type,burger_service_nr,geslachts_naam,familie_betrek_start_datum) VALUES($1,$2,$3,$4,$5,$6,$7) | 9999,0,0,P,000000012,P2,20200101 |
```

Comment:
familie_betrek_start_datum komt alleen voor op een persoon met persoon_type '1' of '2', niet op de persoon (persoon_type='P') zelf

Reply:
wanneer de stap het heeft over 'gewijzigd' moet je een nieuwe insert doen. De eerste situatie(s) van de ouder krijgt/krijgen dan een opgehoogd volg_nr, en de insert krijgt volg_nr 0 met overige gegevens van de laatste situatie plus de opgegeven wijziging.

Reply:
als familie_betrek_start_datum alleen voorkomt op persoon_type=1 dan geld dit dus als correctie binnen de gegevensgroep van het kind. Is dit dan wat er moet gebeuren? zie commit [f988b23](https://github.com/BRP-API/brp-api-gezag/pull/97/commits/f988b230120949d79d012163cbcb0d7f858458dc)